src/handlers/ResponseHandlers.ts

```diff
@@ -57,8 +89,24 @@ const getErrorPropsFromAliases = (responseJson: object, aliasList: string[]): st
     return null;
 };
 
-const defaultResponseHandlers = Object.freeze([noContent, success, error]);
-export const ResponseHandlers = Object.freeze({noContent, success, successBlob, error});
+const defaultResponseHandlers = Object.freeze([
+    noContent,
+    success,
+    blockedByWAFHandler,
+    badGatewayHandler,
+    genericHTMLErrorHandler,
```

Comment:
[Nitpick]: We should aim to follow the same naming pattern as the other handlers, that is without specifying `Handler` in the name since they are all handlers anyway.

```suggestion
    blockedByWAF,
    badGateway,
    htmlError,
```

Reply:
Thanks! Its fixed :)